src/main.c

```diff
@@ -1879,15 +1879,15 @@ int main(int argc, char **argv)
     glBindTexture(GL_TEXTURE_2D, texture);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
-    load_png_texture("textures/texture.png");
+    load_png_texture("/home/user/Remainders-Craft/textures/texture.png");
```

Comment:
This works when I run it, but it could break if someone wants to put the repo in another directory. If there's a simple way to find the absolute path without hard-coding it, that would be preferred.